mlocal/frags/Makefile.stub

```diff
@@ -87,4 +87,53 @@ vendor-check:
 	@echo " VENDOR CHECK"
 	$(V)dep check
 
+.PHONY: uninstall
+uninstall:
+	@echo "INSTALLED PREFIX:  $(PREFIX)"
+	@echo " UNINSTALL:  $(DESTDIR)$(SYSCONFDIR)/singularity/actions/exec"
+	@-rm -fd $(DESTDIR)$(SYSCONFDIR)/singularity/actions/exec
```

Comment:
Note that "DESTDIR" is supplied by the user, so we need to handle that with care.

Please add quotes around the filename:

    @-rm -fd "$(DESTDIR)$(SYSCONFDIR)/singularity/actions/exec"

This seems to be a problem in general with this makefile, but since this deleting files, I think we need to be extra careful.

Also, nitpickig: if you are passing -f, why would rm ever fail? (the - between @ and rm).

Also, more nitpicking: after running this command it's unlikely that the directory will be empty (-d). I think it's better if you explicitly try to remove the directory after we remove everything we know it's in there: `rmdir --ignore-fail-on-non-empty ...`